packages/story-editor/src/components/mediaPicker/useMediaPicker.js

```diff
@@ -84,7 +87,10 @@ export default function useMediaPicker({
       // Race condition concern: the video content is not guaranteed to be
       // available in this callback. For the video poster insertion, please check: packages/story-editor/src/components/library/panes/media/local/mediaPane.js
       wp.Uploader.prototype.success = ({ attributes }) => {
-        updateMedia(attributes.id, { media_source: 'editor' });
+        updateMedia(attributes.id, {
+          media_source: 'editor',
+          alt_text: getAltText(attributes),
```

Comment:
```suggestion
          alt_text: attributes.alt || attributes.title,
```

No need for a `getAltText` helper